api-gateway/graphql-handler/gradle/docker.gradle.kts

```diff
@@ -0,0 +1,95 @@
+/*

```

Comment:
instead of commiting this file, you can include raw github file in `build.gradle.kts`